README.md

```diff
@@ -8,6 +8,8 @@ The project uses a poetry to isolate package dependencies. To get poetry running
 poetry install
 ```
 
+After this create a `.env` file in the `todo_app` folder that matches the `.env.template`. The current app is only supported running on Linux
```

Comment:
It would be nice to extend this a little for the Oauth stuff, e.g. advising a new user how they can acquire those variables by registering an app with Github